website/content/docs/v0.5/Getting Started/prereq-cli-tools.md

```diff
@@ -39,12 +39,12 @@ The main article for installing `clusterctl` can be found
 
 ```bash
 sudo curl -Lo /usr/local/bin/clusterctl \
-  "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.4/clusterctl-$(uname -s | tr '[:upper:]' '[:lower:]')-amd64" \
+  "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.4/clusterctl-$(uname -s | tr '[:upper:]' '[:lower:]')-amd64" \
```

Comment:
`1.1.1` is the latest

Reply:
we didn't test with that, so I thought to be on the safe side :shrug: 

we can bump it, I didn't want to bump too much for the release, as we tested with 1.0.x